client.go

```diff
@@ -50,6 +50,9 @@ type PublishResult struct {
 	Offset    uint64
 }
 
+type Subscriber = func(ctx context.Context, payload []byte, contentType string) error
```

Comment:
maybe this type should be called `EventHandler` or simply `Handler`?

Reply:
sounds good. renamed to EventHandler.